### DISCUSSION TITLE
[Feature] Support for MySql/Mariadb

### DISCUSSION THREAD

**Discussion by arana198** [immich-app/immich#3674] [Feature Request]
### The feature

Would be nice to add support for MySQL/Mariadb 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by samip5** [comment]
Immich uses Postgres specific queries, not worth it as a result.

**Comment by arana198** [comment]
It's unfortunate that I rely on High Availability (HA) MariaDB, which means I won't be able to utilize immich. Additionally, managing two separate databases isn't something I'm keen on.

I hope mariadb is integrated in the future.

**Reply by samip5** [reply]
Same can be said about Postgres with HA. ;)
